L1Trigger/L1TTwinMux/src/L1TTwinMuxAlgorithm.cc

```diff
@@ -109,32 +116,42 @@ void L1TwinMuxAlgortithm::run(
       for (sector=0;sector<12; sector++ ){
         for (station=1; station<=4; station++){
 
-          dtts1=0; dtts2=0; rpcts1=0;
-
-          dtts1 = phiDigiscp.chPhiSegm1(wheel,station,sector,bx);
-          dtts2 = phiDigiscp.chPhiSegm2(wheel,station,sector,bx);
-          rpcts1 = rpcPhiDigis.chPhiSegm1(wheel,station,sector,bx);
-
-            if(!onlyRPC) {
-              if(!dtts1 && !dtts2 && !rpcts1 ) continue;
+         // dtts1=0; dtts2=0; rpcts1=0;
+
+         // dtts1 = phiDigiscp.chPhiSegm1(wheel,station,sector,bx);
+         // dtts2 = phiDigiscp.chPhiSegm2(wheel,station,sector,bx);
+         // rpcts1 = rpcPhiDigis.chPhiSegm1(wheel,station,sector,bx);
+           dtts1.reset(new L1MuDTChambPhDigi(0,0,0,0,0,0,7,0,0,0));
```

Comment:
This will probably give a bad performance since new L1MuDTChampPhDigis are constructed all the time.
Better to keep the raw pointers in this case.